docs/2021/minerva/index.md

```diff
@@ -9,19 +9,28 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 
 Why do we need to create OSS Dataset - 
 
-1. IRREGULARITY IN THE SIZE OF LICENSE TEXTS - The license texts are of different sizes with huge differences in terms of keywords count. Longer license texts contain most of the unique keywords when compared against the uniqueness of keywords in the smaller license texts. Dynamic Time Warping has the capability to tackle a varying length of array of word-embeddings of different license files.
+1. IRREGULARITY IN THE SIZE OF LICENSE TEXTS - The license texts are of different sizes with huge differences in terms of keywords count. Longer license texts contain most of the unique keywords when compared against the uniqueness of keywords in the smaller license texts.
 
-2. LICENSE TEXTS ARE DIFFERENT THAN TRADITIONAL TEXT CORPORA - Usually, in the traditional corpus, the documents are different to some extent that differentiate them with each other whereas, in license texts, most of the tokens are similar in the majority of the texts. There is a very slight difference in the use of these tokens to create a license statement. The keywords used in these license texts can be found in almost all of them with a slight variation since they all are open-source licenses stating open source softwares and underlying permissions. These similarities in licenses make them tough to be differentiated by any traditional information retrieval algorithm.
+2. DIFFERENT THAN TRADITIONAL TEXT CORPORA - In licenses most of the tokens are similar and the keywords used can be found in almost all of them with a slight variation since they all are open-source licenses stating open source softwares and underlying permissions. These similarities in licenses make them tough to be differentiated by any traditional information retrieval algorithm.
 
-So, the idea is to generate SPDX OSS license dataset using FOSSOLOGY NOMOS AGENT STRINGS.in REGEX and other SPDX released Licenses regex using python libraries and NLP algorithms. Now, to proceed with text generation using regex - we don’t need to generate texts from scratch, instead, we can use an existing file as a baseline model for further manipulating and generating texts from those files. Also, we have to specify the length for text generated, because we don't want it to run till infinity. So a seed length will be taken depending on the length of each dataset available in the dataset. Further the idea extends to validation of the generated license files. 
+So, the idea is to generate SPDX OSS license dataset using FOSSOLOGY NOMOS AGENT STRINGS.in REGEX and latest SPDX released Licenses. We can use an existing file as a baseline model for further manipulating and generating texts from those files.
 
 ## What is that I am doing right now?
-![project-overview](https://user-images.githubusercontent.com/50830709/125211400-b2ba8380-e2c3-11eb-9d44-1bab34547f04.JPG)
+![image](https://user-images.githubusercontent.com/50830709/128769487-8f4b2cae-c88f-4540-9d83-897bc7ff2c18.png)
```

Comment:
Please commit all images in the repo itself se we do not loose them in future.
To do so, add them in `static/img/minerva/` folder and then change the MD to `![image](/img/minerva/image-file.png)`

Reply:
Please update the file path.

```suggestion
![image](/img/minerva/project_overview.png)
```